content/assignments/hello-world/_index.md

```diff
@@ -0,0 +1,151 @@
+   * **Windows Users:** Click on the solid green triangle next to the name of the `HelloWorldAutograded` project. You can also press F5 to run the project.
+
+
+1. Visual Studio opens a pane and displays any program output when you run a program. Currently, the program output contains the following:
+   
+   ``` console 
+   &&&
+   System.String
+   ```
+1. You are ready to run the auto-grading tests.
+
+## Running the Auto-grading Tests
+
+The tests provide feedback about your code before you submit your work.  Use the tests to check your work as you complete the assignment.
+
+The tests are unit tests for you to use to check your work.  We call them auto-graded tests instead of unit tests in these projects. 
+
+Let’s learn how to run the tests in Visual Studio.
+
+1. Each assignment will contain a test **project**.  This project contains all of the auto-grading tests and may have more than one test class.  In the Solution Explorer, navigate to HelloTests.cs inside the `Hello.Tests` project. 
+
+   {{% notice blue "Note" "rocket" %}}
+   You will not need to open the `Using.cs` file.  This file connects the testing project to the testing library.  This is not a test class.
+   {{% /notice %}}
+
+   A **test** is a single method in a test file with `[TestMethod]` above it. 
+
+      ```csharp
+      [TestClass]
+      public class AutogradingTests
+      {
+         [TestMethod]
+         public void TestToDemonstrateWhatATestMethodLooksLike()
+         {
+            //test code here
+         }
+      }
+      ```
+
+      A test class may contain many tests. 
+
+
+      ```csharp
+      [TestClass]
+      public class AutogradingTests
+      {
+         [TestMethod]
+         public void TestToDemonstrateWhatATestMethodLooksLike()
+         {
+            //test code here
+         }
+
+         [TestMethod]
+         public void SecondTestToDemonstrateWhatATestMethodLooksLike()
+         {
+            //test code here
+         }
+      }
+   ```
+1. Run all of your tests using the Test Explorer window.
+   1. Mac Users: 
+      1. **View** > **Tests**  
+      1. Select the green triangle on the left of the Test Explorer. 
+      1. Step through the solution until you find the test that failed. 
+      1. [Guide](https://learn.microsoft.com/en-us/visualstudio/mac/testing?view=vsmac-2022#running-tests) to running tests in Visual Studio for Mac
+   1. Windows Users 
+      1. **Test** > **Windows** > **Test Explorer**
+      1. Select the `Run All` icon to run all of your tests. 
+      1. In the Test Explorer, step through your solution until you find the failed test.
+      1. [Guide](https://learn.microsoft.com/en-us/visualstudio/test/run-unit-tests-with-test-explorer?view=vs-2022) to running tests in Visual Studio.
+
+
+1. Visual Studio opens a Test Pane to display test results every time it runs a test. 
```

Comment:
Test Results pane, at the bottom of the window with all the information of what passed and what failed.